### DISCUSSION TITLE
Emma and Hakeem

### DISCUSSION THREAD

**Discussion by lyonhakeem** [birkenkrahe/ai482#11] [Ideas]
[Presentation Link](https://drive.google.com/file/d/1vaVugJ90nhiljaZgGDoRfmajhPiVOOTK/view?usp=sharingl)

**Comment by birkenkrahe** [comment]
HI - just a few comments, since because of my Germany trip I only have a couple of hours left to turn in the grades...enjoyed the presentation! Especially because you took the "product development" challenge seriously - there's a hidden issue here, namely why aren't there more cross-over applications like the one you suggested? It's possible that the answer is not technical but commercial. I'll be thinking about that, especially since "sports analytics" may become one of my future research topics.

Otherwise:
   * Good side-by-side presentation
   * Cross-over app suggestion (best of both worlds)
   * Like the drawings!
   * Product development was interesting
   * References consistent and complete
